README.md

```diff
@@ -83,7 +83,7 @@ This allows addons built with it to run with Node.js versions which support the
 **However** the node-addon-api support model is to support only the active LTS Node.js versions. This means that
 every year there will be a new major which drops support for the Node.js LTS version which has gone out of service.
 
-The oldest Node.js version supported by the current version of node-addon-api is Node.js 14.x.
+The oldest Node.js version supported by the current version of node-addon-api is Node.js 18.x.
```

Comment:
Should it be 16.x?

Reply:
Addressed in [480ca9c](https://github.com/nodejs/node-addon-api/pull/1324/commits/480ca9c81cdf5faef4ceae3cf34121f1bc066c91)